websrc/source/documents/tutorials/container-101.md

```diff
@@ -321,24 +324,24 @@ There are two main container networking models discussed within the community.
 
 #### Docker libnetwork - Container Network Model (CNM)
 
-CNM (Container Network Model) is Docker's libnetwork network model for containers
-- An endpoint is container's interface into a network
-- A network is collection of arbitrary endpoints
-- A container can belong to multiple endpoints (and therefore multiple networks)
-- CNM allows for co-existence of multiple drivers, with a network managed by one driver
-- Provides Driver APIs for IPAM and Endpoint creation/deletion
-- IPAM Driver APIs: Create/Delete Pool, Allocate/Free IP Address
-- Network Driver APIs: Network Create/Delete, Endpoint Create/Delete/Join/Leave
+CNM (Container Network Model) is Docker's libnetwork network model for containers  
```

Comment:
What's up with the extra whitespace here?  Was this automatically added by your editor?

You can edit this file and amend the commit and force push it (good opportunity to practice 😄)